css/menu.css

```diff
@@ -46,6 +46,9 @@ body.disableScroll {
 [data-has-notch] .fl-bottom-bar-menu-holder.expanded,
 .fl-bottom-bar-menu-holder.expanded ul {
   transform: translate3d(0, 0, 0);
+  overflow-y: scroll;
+  right: -30px;
```

Comment:
Why is there a `right: -30px`?

Reply:
I removed it from the rules instead I added new logic in the menu.js